packages/core/core/src/loadParcelConfig.js

```diff
@@ -42,48 +43,50 @@ export default async function loadParcelConfig(
   return parcelConfig;
 }
 
-export async function resolve(filePath: FilePath): Promise<?Config> {
+export async function resolve(filePath: FilePath) {
```

Comment:
This is a personal thing and we should probably come to an understanding across Parcel, but I prefer to explicitly type function return types for exported functions.